nevergrad/optimization/multiobjective/core.py

```diff
@@ -150,7 +150,7 @@ def pareto_front(
         size:  int (optional)
             if provided, selects a subset of the full pareto front with the given maximum size
         subset: str
-            method for selecting the subset ("random, "loss-covering", "EPS", "domain-covering", "hypervolume")
+            method for selecting the subset ("random, "loss-covering", "EPS", "EPS2", "domain-covering", "hypervolume")
             EPS is the epsilon indicator described e.g.
                 here: https://hal.archives-ouvertes.fr/hal-01159961v2/document
```

Comment:
please describe what is EPS2, or possibly rename it, if it's a min-max stuff I guess there could be a more accurate name

Reply:
- the old EPS is now removed
- EPS2 is therefore now termed EPS